backend/CircleForms/Services/Database/RedisCacheRepository.cs

```diff
@@ -67,9 +68,17 @@ public async Task<bool> UserExists(string id)
         return await _redis.SetContainsAsync(_userIds, id);
     }
 
-    public async Task AddUserToUserIds(string id)
+    public async Task AddUser(User user)
     {
-        await _redis.SetAddAsync(_userIds, id);
+        await _redis.SetAddAsync(_userIds, user.ID);
+        var cachedUser = _mapper.Map<User, UserMinimalRedis>(user);
+        await _redis.StringSetAsync($"user:{user.ID}", JsonConvert.SerializeObject(cachedUser));
```

Comment:
Use `.ToUserId()`? 
`user.ID.ToString().ToUserId()` will look rather ugly, but at least it will make sure you can't accidentally forget about key usage here